src/java/org/apache/cassandra/cql3/statements/schema/AlterTableStatement.java

```diff
@@ -739,9 +739,9 @@ public KeyspaceMetadata apply(Epoch epoch, KeyspaceMetadata keyspace, TableMetad
     public static class AlterConstraints extends AlterTableStatement
     {
         final ColumnIdentifier columnName;
-        final ColumnConstraints constraints;
+        final ColumnConstraints.Raw constraints;
 
-        AlterConstraints(String keyspaceName, String tableName, boolean ifTableExists, ColumnIdentifier columnName, ColumnConstraints constraints)
+        AlterConstraints(String keyspaceName, String tableName, boolean ifTableExists, ColumnIdentifier columnName, ColumnConstraints.Raw constraints)
```

Comment:
This drops the `IF EXISTS` for the column, e.g.:
```
CREATE TABLE ks.t1 (k int PRIMARY KEY, a int);
ALTER TABLE ks.t1 ALTER IF EXISTS b CHECK b > 0;
```
should not error because `b` doesn't exist

Reply:
Added a test and it works:
```
@Test
    public void testCreateTableAddConstraintWithIfExists() throws Throwable
    {
        createTable("CREATE TABLE %s (pk int, ck1 int, ck2 int, v int, PRIMARY KEY ((pk),ck1, ck2)) WITH CLUSTERING ORDER BY (ck1 ASC);");
        execute("ALTER TABLE %s ALTER IF EXISTS foo CHECK foo < 100");
    }
```

Reply:
It is also silent when the `IF EXISTS` is not supplied though (i.e. when it _should_ error). You should probably do something similar to the other implementations of `AlterTableStatement`:
```
            ColumnMetadata currentColumn = table.getColumn(column);
            if (null == currentColumn) {
                if (!ifExists)
                    throw ire("Column %s was not found in table '%s'", column, table);
                return;
            }
```